browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -26,6 +25,11 @@
 #include "components/bookmarks/common/bookmark_pref_names.h"
 #include "components/prefs/pref_service.h"
 
+#if BUILDFLAG(ENABLE_SPEEDREADER)
+#include "brave/components/speedreader/features.h"
```

Comment:
is there a dep for `//brave/components/speedreader`?

Reply:
in `brave/browser/ui/BUILD.gn`. This doesn't change deps from before, only adds buildflag condition that wasn't there for pref name

Reply:
oh right, this part was moved, but it does appear to be missing the buildflag dep that is required for https://github.com/brave/brave-core/pull/5529/files#diff-3d72382b1c4fba0ded2fc0100530ff54R18

Reply:
should be fine now